src/App.js

```diff
@@ -18,6 +18,14 @@ function App() {
         >
           Learn React
         </a>
+        <div className="App-Logo2">
+          <p>Logo</p>
+        </div>
+        <div className="lowerLeft">
```

Comment:
Not that I suggest changing it, because done code is highly valuable, but in the future I would stay aware from names that suggest locations.  Locations change, sometimes even based on the screen size, which could make this name confusing.